### PR TITLE
Fix incorrect problem alias

### LINKED ISSUE #1851: Incorrect problem alias
In `/arena/CONTEST_ALIAS#ranking`
Strange contest alias in scoreboard when there are more than 26 problems in a contest

## Current behaviour
Contest become character `[`, `\`, `]`, `^`, etc. This was caused by ascii incremental

## Expected behaviour
Alias bigger than letter Z should be shown as AA, AB, AC, etc

## Possible solution
Change this vue
https://github.com/omegaup/omegaup/blob/master/frontend/www/js/omegaup/components/arena/Scoreboard.vue#L12 

Screenshots :
screenshot from 2018-03-19 08-55-17
screenshot from 2018-03-19 08-55-31

### PR DESCRIPTION
**Pull request by SpaceWhite**
Fix #1851

### CODE REVIEW

**Comment by SpaceWhite** [comment]
My bad I didn't push it with vagrant

**Comment by SpaceWhite** [comment]
@lhchavez Here you go

File: frontend/www/js/omegaup/components/arena/Scoreboard.vue [hunk 1]
```diff
@@ -83,6 +83,15 @@ export default {
       return (idx < this.scoreboardColors.length) ? this.scoreboardColors[idx] :
                                                     '';
     },
+    columnName: function(idx) {
+      var name = String.fromCharCode("A".charCodeAt(0) + idx % 26);
+      while (idx >= 26) {
+        idx = (idx / 26) >> 0;
```
Comment (lhchavez) [inline frontend/www/js/omegaup/components/arena/Scoreboard.vue]: btw, why is this `>> 0` needed? are you trying to coerce this expression as an integer? If so, can you use what asm.js does?

```javascript
idx = (idx / 26)|0;
```

Comment (SpaceWhite) [reply frontend/www/js/omegaup/components/arena/Scoreboard.vue]: Yes, I'm trying to floor it. Is it bad to write like that? @lhchavez

File: frontend/www/js/omegaup/components/arena/Scoreboard.vue [hunk 2]
```diff
@@ -83,6 +83,15 @@ export default {
       return (idx < this.scoreboardColors.length) ? this.scoreboardColors[idx] :
                                                     '';
     },
+    columnName: function(idx) {
```
Comment (pabloaguilar) [inline frontend/www/js/omegaup/components/arena/Scoreboard.vue]: Can you move this to frontend/www/js/omegaup/ui.js, please?
It's not tied to Scoreboard and might come in useful in other places

Comment (SpaceWhite) [reply frontend/www/js/omegaup/components/arena/Scoreboard.vue]: @pabloaguilar added

**Pull request merged**